src/main/java/movie/storage/model/Movie.java

```diff
@@ -16,4 +14,40 @@
     private Long id;
     private String title;
     private String description;
+
+    public Movie(String title, String description) {
+        this.title = title;
+        this.description = description;
+    }
+
+    public Movie(Long id) {
+        this.id = id;
+    }
+
+    public Movie() {
+    }
```

Comment:
in general, too many constructors are not ok
let's better use setters

Reply:
Agree